frontend/src/scenes/insights/InsightDisplayConfig.tsx

```diff
@@ -83,11 +86,13 @@ export function InsightDisplayConfig({ filters, activeView, disableTable }: Insi
     const showPathOptions = activeView === InsightType.PATHS
     const { featureFlags } = useValues(featureFlagLogic)
 
+    const { setFilters } = useActions(insightLogic)
+
     return (
         <div className="display-config-inner">
```

Comment:
Whilst you're here _if you want_ we could delete the other css code in favor of 
```suggestion
        <div className="flex justify-between items-center flex-wrap">
```